test/e2e/networks/ci.toml

```diff
@@ -11,6 +11,7 @@ process_proposal_delay = "100ms"
 check_tx_delay = "0ms"
 # The most common case (e.g. Cosmos SDK-based chains).
 abci_protocol = "builtin"
+prometheus = true
```

Comment:
So we are running prometheus enabled nodes in our CI as well? Is that going to cause too much noise?

Reply:
Prometheus is enable from now on, but no server set up. Although it might seem a bit silly, it would have caught the two panics I saw when I started the DO testnet.